versions.yaml

```diff
@@ -18,17 +18,16 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 versions:
-  - dir: '2/2.32'
+  - dir: '2/2.46.2'
     repo: jenkins2
     tags:
-      - 2.32
-      - 2.32.2
+      - 2.46.2
```

Comment:
Didn't address Wojciech's comment: We should also add tag `2` to this version for completeness as well, even though Launcher doesn't use tags other than `latest`.